### ISSUE TITLE
End-of-line comments are not parsed correctly

### ISSUE THREAD

**Issue by ziotom78** [avakar/pytoml#7]
The following example is taken from the current (0.4) TOML specification (https://github.com/toml-lang/toml#user-content-comment):

`````toml
    # This is a full-line comment
    key = "value" # This is a comment at the end of a line
`````

Parsing it with `pytoml` produces an error near the comment on the second line:

`````python
import pytoml
pytoml.loads("""
# This is a full-line comment
key = "value" # This is a comment at the end of a line
""")
`````

The error is:

    TomlError: <string>(2, 15): msg

**Comment by avakar** [comment]
Whoops, thanks for the report.

**Comment by avakar** [comment]
Hmm, I couldn't repro this. Are you sure you have the newest version installed?

    c:\devel\checkouts>python
    ActivePython 2.7.8.10 (ActiveState Software Inc.) based on
    Python 2.7.8 (default, Jul  2 2014, 19:50:44) [MSC v.1500 32 bit (Intel)] on win32
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import pytoml
    >>> pytoml.loads("""
    ... # This is a full-line comment
    ... key = "value" # This is a comment at the end of a line
    ... """)
    {u'key': u'value'}
    >>> import pkg_resources; pkg_resources.get_distribution("pytoml").version
    '0.1.4'

**Comment by ziotom78** [comment]
I'm sorry, I tried to run again my example and indeed it worked flawlessly. The fact is that I got a number of weird errors on an apparently legitimate TOML file, and I believed that the problem was in the end-of-line comment. After more tests, I think the problem lies in the way floating point values are parsed: I have therefore opened another issue, see here: https://github.com/avakar/pytoml/issues/9.
